logrotate: add page (#9055)
tldr-pages/tldr: pages/linux/logrotate.md

# logrotate

> Rotates, compresses, and mails system logs.

> > More information: <https://manned.org/logrotate.8>.

Reviewer: ```suggestion
> More information: <https://manned.org/logrotate>.
```

> - Trigger a run of logrotate manually:

Reviewer 2: ```suggestion
- Trigger a run manually:
```
I think this would be better.

> - Force logrotate to run outside of normal interval:

Reviewer 3: This sounds a little confusing, when you don't know that it runs in intervals. How about:
```suggestion
- Trigger a run of  logrotate manually:
```

`logrotate {{path/to/logrotate.conf}} --force`

> - Set mail binary for logrotate to Mutt:

Reviewer 3: Is Mutt something usually used with logrotate? If it's just a random example, we should probably keep it more general with `/usr/bin/mail_binary`.
```suggestion
- Set the mail binary for logrotate to Mutt:
```

> - Run logrotate using a specific command to mail reports:

Reviewer 2: ```suggestion
- Run using a specific command to mail reports:
```

> - Run logrotate using a specific mail_binary to send reports:

Reviewer: Perhaps this better
```suggestion
- Run logrotate using a specific command to mail reports:
```

> `logrotate {{path/to/logrotate.conf}} --mail /usr/bin/mutt`

Reviewer 3: Even though it's a specific example, it's still a variable argument:
```suggestion
`logrotate {{path/to/logrotate.conf}} --mail {{/usr/bin/mutt}}`
```

> `logrotate {{path/to/logrotate.conf}} --mail {{/usr/bin/mail_binary}}`

Reviewer: ```suggestion
`logrotate {{path/to/logrotate.conf}} --mail {{path/to/mail_command}}`
```

> - Run logrotate without lock or state file:

Reviewer: ```suggestion
- Run without using a state (lock) file:
```

Author: I like this a lot better, thank you for the suggestion!

> `logrotate {{path/to/logrotate.conf}} --state /dev/null`

Reviewer 3: ```suggestion
`logrotate {{path/to/logrotate.conf}} --state {{/dev/null}}`
```

Reviewer: No, the use of `/dev/null` is a specific usage scenario.

> - Skip lock file check:

Reviewer: ```suggestion
- Run skipping lock file check:
```

> - Run skipping lock file check:

Reviewer 3: Wouldn't this imply `--force`, since it's run outside the default interval?
```suggestion
- Run logrotate, skipping the lock file check:
```

> - Run logrotate and skip the state (lock) file check:

Reviewer 2: ```suggestion
- Run and skip the state (lock) file check:
```

`logrotate {{path/to/logrotate.conf}} --skip-state-lock`

> - Log instances of logrotate:

Reviewer 3: I'm not really sure what this does. Could you elaborate?

> - Tell `logrotate` to log verbose output into the log_file:

Reviewer 2: ```suggestion
- Tell `logrotate` to log verbose output into the `log_file`:
```
Just a small suggestion

> - Tell logrotate to log verbose output into the log_file:

Reviewer 2: ```suggestion
- Tell `logrotate` to log verbose output into the log_file:
```
